### PR TITLE
bump to 0.4

### PR DESCRIPTION
**Pull request by bernt-matthias**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by bernt-matthias** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,37 +1,50 @@
...
+    - pandamesh >=0.2.1
+    - ogs >=6.5.3
+    - typeguard >=4.0.0
+    - lxml >=4.6.0
 
 test:
   imports:
     - ogstools
   commands:
     - pip check
```
Comment (bernt-matthias) [inline recipe/meta.yaml]: If `ogs` is still not found maybe try 

```suggestion
    - pip list
    - pip check
```

**Comment by bernt-matthias** [comment]
Are the OSX pipelines needed, since the package is `noarch: python`?

**Review by bilke — APPROVED** [review]
Ready from my side. @bernt-matthias Ready for you too?

**Comment by bernt-matthias** [comment]
Looks good to me.

**Pull request merged**